pwa/src/apiService/resources/mapping.ts

```diff
@@ -47,4 +47,15 @@ export default class Mapping {
     });
     return data;
   };
+
+  public testSimpleMapping = async (variables: { payload: any; id: string }): Promise<any> => {
```

Comment:
I don't think we have "complex" mappings (that can be tested)? Maybe `testMapping` would be a better name.